env0/resource_environment_test.go

```diff
@@ -193,6 +215,84 @@ func TestUnitEnvironmentResource(t *testing.T) {
 			})
 		})
 
+		t.Run("vcs pr comments enabled", func(t *testing.T) {
```

Comment:
Can we specifically also test the scenario where `vcs_commands_alias` is sent without specifying `vcs_pr_comments_enabled` ?

Reply:
I don't think it really matters in these type of tests (since the tests are mocked).
I was asked not handle this use-case in the provider. Instead the backend will handle it.

I we already have a harness test passing an alias without vcs_pr_comments_enabled.

```
resource "env0_environment" "example" {
  depends_on    = [env0_template_project_assignment.assignment]
  force_destroy = true
  name          = "environment-${random_string.random.result}"
  project_id    = env0_project.test_project.id
  template_id   = env0_template.template.id
  configuration {
    name  = "environment configuration variable"
    value = "value"
  }
  approve_plan_automatically = true
  revision                   = "master"
  vcs_commands_alias         = "alias"
  drift_detection_cron       = var.second_run ? "*/5 * * * *" : "*/10 * * * *"
}
```